src/applications/personalization/profile360-beta/components/ProfileView.jsx

```diff
@@ -0,0 +1,175 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import DowntimeNotification, {
+  externalServices,
+  externalServiceStatus,
+} from 'platform/monitoring/DowntimeNotification';
+import DowntimeApproaching from 'platform/monitoring/DowntimeNotification/components/DowntimeApproaching';
+import recordEvent from 'platform/monitoring/record-event';
+
+import Vet360TransactionReporter from 'vet360/containers/TransactionReporter';
+
+import Hero from '@profile360/components/Hero';
```

Comment:
The `@profile360` alias resolves to `/src/applications/personalization/profile360`